ext/node/polyfills/worker_threads.ts

```diff
@@ -281,6 +282,7 @@ class NodeWorker extends EventEmitter {
       op_host_terminate_worker(this.#id);
     }
     this.emit("exit", 1);
+    return PromiseResolve(0);
```

Comment:
Shouldn't it return the same code as `exit`?

Reply:
Need to compare with nice. Haven't checked what they emit